### PR TITLE
sort/format pypi_formula_mappings.json

### PR DESCRIPTION
**Pull request by singingwolfboy**
I used [this online JSON sorting tool](https://www.coderstool.com/json-sort) to sort the contents of `pypi_formula_mappings.json`, and then reformatted it with [`prettier`](https://prettier.io). The data remains unchanged.

### CODE REVIEW

File: pypi_formula_mappings.json [hunk 1]
```diff
@@ -459,22 +600,41 @@
...
+      "attrs",
+      "pyrsistent"
+    ]
   },
   "sphinx-doc": {
     "extra_packages": ["numpydoc"]
   },
   "sqlfluff": {
-    "exclude_packages": ["PyYAML"]
+    "exclude_packages": ["PyYAML", "typing-extensions"]
```
Comment (SMillerDev) [inline pypi_formula_mappings.json:637]: Definitely not unchanged

**Review by SMillerDev — CHANGES_REQUESTED** [review]
What is the benefit of making the file longer?

File: pypi_formula_mappings.json (hunk 1 above)
Comment (singingwolfboy) [reply pypi_formula_mappings.json:637]: Oops, I did this after making #112865. I'll revert this change.

**Comment by singingwolfboy** [comment]
> What is the benefit of making the file longer?

As I said, I reformatted the file using `prettier`, which generally tries to optimize for readability rather than file length. However, if this does not seem like a good change, then we can just close this pull request.

**Comment by SMillerDev** [comment]
> As I said, I reformatted the file using prettier, which generally tries to optimize for readability rather than file length.

It's not something a lot of people will ever read, but it is something a lot of people will download. I don't think that's worth it.

**Pull request closed without merging**